crates/sui-graphql-rpc/src/server/builder.rs

```diff
@@ -98,3 +108,78 @@ async fn graphiql() -> impl axum::response::IntoResponse {
             .finish(),
     )
 }
+
+#[cfg(test)]
+mod tests {
```

Comment:
The mix of `use` vs fully qualified names is adding a bunch of line noise.  Can you:

- Add a single `use super::*;` to pull in the parent module's imports (this should cover most things).
- Import other names once at the top of the module, rather than within the function, or fully qualified at the reference site?